staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -309,12 +323,10 @@ func (s *APIAggregator) AddAPIService(apiService *v1.APIService) error {
 
 	// register the proxy handler
 	proxyHandler := &proxyHandler{
-		localDelegate:   s.delegateHandler,
-		proxyClientCert: s.proxyClientCert,
-		proxyClientKey:  s.proxyClientKey,
-		proxyTransport:  s.proxyTransport,
-		serviceResolver: s.serviceResolver,
-		egressSelector:  s.egressSelector,
```

Comment:
this change surprises me.  what's going on?

Reply:
what exactly? we removed `proxyClientCert` and `proxyClientKey` but we added `proxyCurrentCertKeyContent`

Reply:
indeed, it was missing - thx.